scripts/csv_colours.py

```diff
@@ -0,0 +1,59 @@
+from vectordatasource.colour import parse_colour
+from vectordatasource.transform import Palette
+from optparse import OptionParser
+import sys
+import csv
+import yaml
+
+
+def to_hex(colour):
+    return "#%02x%02x%02x" % tuple(colour)
+
+
+yaml_config_file = None
+output_file = sys.stdout
+mz_colour_key = 'MZ colour'
```

Comment:
As a general point, we use the `mz_` prefix all over the place, so it doesn't make much sense to use tilezen here rather than `mz_`. I've changed it to `output_colour_key` in ed2f38c. And if we want to do a more general `mz -> tz` change, then we should do that as a follow-up issue.